lib/response_comparator.rb

```diff
@@ -25,6 +25,13 @@ def self.quick_comparison(primary_response, secondary_response)
     }
   end
 
+  def self.full_comparison?(comparison, full_pct)
+    r = rand(99)
+    full = (full_pct == 100 || r < full_pct)
```

Comment:
If the above comment is taken into account, could this `full_pct == 100` be removed so the entire method would be 🤔 

```ruby
def self.full_comparison?(comparison, full_pct)
    r = rand(100) # between 0 - 99
    comparison.merge!(sample_percent: full_pct, r:)
    full
  end
```

Reply:
good call, simplified as per your suggestion - thanks!